server/plugin/command.go

```diff
@@ -470,7 +470,14 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 		}
 
 		privateAllowed := false
-		if len(parameters) > 0 {
+		if p.getConfiguration().SetPrivateAsDefault && len(parameters) > 0 {
+			p.postCommandResponse(args, fmt.Sprintf("Unknown command `%v`. Do you meant `/github connect`?", args.Command))
+			return &model.CommandResponse{}, nil
```

Comment:
Do we want to allow connecting with only public repos once private repos are enabled by default?

Reply:
No, it should connect with both public and private repos without writing private in slash command for private repos.